### PR TITLE
Add k4edm4hep2lcioconv recipe

### PR DESCRIPTION
**Pull request by fdplacido**
BEGINRELEASENOTES
- Add recipe for [k4edm4hep2lcioconv](https://github.com/key4hep/k4EDM4hep2LcioConv/)

ENDRELEASENOTES

### CODE REVIEW

**Review by vvolkl — APPROVED** [review]

**Comment by vvolkl** [comment]
Thanks, maybe a good idea to tag a first version already so it can be included properly in the stack? 

And, if you want the HEAD of this to be built in the nightlies, add 

```
k4_add_latest_commit('k4edm4hep2lcioconv', 'key4hep/k4edm4hep2lcioconv')
```
here:

https://github.com/key4hep/key4hep-spack/blob/release/scripts/fetch_nightly_versions.py#L61

**Comment by vvolkl** [comment]
Waiting for a preliminary release before merging.

**Comment by fdplacido** [comment]
> Waiting for a preliminary release before merging.

I created the release and updated the recipe

**Pull request merged**
